cwms-data-api/src/test/java/fixtures/TestAccounts.java

```diff
@@ -35,6 +35,7 @@ public enum KeyUser {
         NONE("none",null,null,null,null, null), // Used for annotations
         GUEST("guest",null,null,null, null, null), // USED as marker label for tests
         SPK_NORMAL("l2hectest","l2hectest","1234567890","l2userkey","ATotallyRandomStringL2hectest","SPK", "CWMS Users", ApiServlet.CAC_USER),
+        SPK_NEW_USER("l2hectest4","l2hectest4","12345678904","l2userkey4","ATotallyRandomStringL2hectest4","SPK", "CWMS Users", ApiServlet.CAC_USER),
```

Comment:
I don't really love the SPK_NEW_USER as the name.  what happens if you want another new user? It would be SPK_NEWER_USER?